enterprise/cypher/acceptance-spec-suite/src/test/scala/org/neo4j/internal/cypher/acceptance/TemporalFunctionsAcceptanceTest.scala

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2002-2018 "Neo Technology,"
+ * Network Engine for Objects in Lund AB [http://neotechnology.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Neo4j is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+package org.neo4j.internal.cypher.acceptance
+
+import org.neo4j.cypher.{FakeClock, ExecutionEngineFunSuite}
+import org.neo4j.internal.cypher.acceptance.CypherComparisonSupport._
+import org.neo4j.values.storable.DateTimeValue
+
+class TemporalFunctionsAcceptanceTest extends ExecutionEngineFunSuite with CypherComparisonSupport with FakeClock {
+
+  val supported = (Configs.Version3_4 + Configs.Version3_3 + Configs.Version3_1) - Configs.Compiled
+
+  test("should get current default datetime") {
+    val result = executeWith(supported, "RETURN datetime() as now")
+
+    val now = single(result.columnAs[DateTimeValue]("now"))
+
+    now shouldBe a[DateTimeValue]
+  }
+
+  test("should get current 'now' datetime") {
+    val result = executeWith(supported, "RETURN datetime({when:'now'}) as now")
```

Comment:
This looks weird to me. I get using maps for mult-argument procedures, to avoid using positional arguments. For single argument procs, I think it's unnecessary to be inconsistent with most other procedures and functions we have.

Reply:
Please see [CIP2015-08-06](https://github.com/thobe/openCypher/blob/date-time/cip/CIP2015-08-06-date-time.adoc), the map is needed here to distinguish from the different overloads of the function. It looks weird because the other tests have not been written yet.

Reply:
But your comment got me thinking. So we discussed it and came up with what might be a better way to do this particular thing: https://github.com/opencypher/openCypher/pull/15/commits/5bb14895b95e40a0e3769c9120c9e3b161787227